tests/test.display.js

```diff
@@ -353,15 +353,12 @@ describe('Display/Canvas Helper', function () {
             let large_checked_data = new Uint8Array(16*16*4);
             display.resize(16, 16);
 
-            for (let y = 0;y < 16;y++) {
-                for (let x = 0;x < 16;x++) {
-                    let pixel;
-                    if ((x < 4) && (y < 4)) {
+            for (let y = 0; y < 16; y++) {
+                for (let x = 0; x < 16; x++) {
+                    const pixel = ((x < 4) && (y < 4))
```

Comment:
this is a fairly long use of the ternary operator.  It's clearer as an if statement

Reply:
This is just to keep the variable as constant instead of keeping it mutable. As mentioned before const should be preferred since they make the code more solid in general.

But I can accept that this might be a bit big ternary operator.